tmux-run_local.sh

```diff
@@ -1,12 +1,26 @@
 unset PROMPT_COMMAND
 
+sync=false
+
+for arg in "$@"; do
+  if [[ $arg == "--sync" || $arg == "-s" ]]; then
+    sync=true
+  fi
+done
+
+echo $sync >> sync.txt
+
 tmux set -g pane-border-status top
 tmux set -g mouse on
 tmux set -g pane-border-format "#{pane_title}"
 
-tmux split-window -fh "tmux select-pane -T \"Web Backend\"; ./start_web-run_local.sh 2>&1 | tee logs/web.log"
+tmux split-window -fh "tmux select-pane -T \"Web Backend\"; ./start_web-run_local.sh || read"
```

Comment:
Why use the `read` command here?

Reply:
To prevent the pane from dying on crash, it only runs if the command stops with an error.